test/e2e/node_drain.go

```diff
@@ -432,7 +462,35 @@ func NodeDrainTimeoutSpec(ctx context.Context, inputGetter func() NodeDrainTimeo
 			WaitForMachineDeployments: input.E2EConfig.GetIntervals(specName, "wait-worker-nodes"),
 		})
 
-		By("Verify scale down succeeded because Node drains were unblocked")
+		if input.VerifyNodeVolumeDetach {
```

Comment:
might be we should validate that both VerifyNodeVolumeDetach and UnblockNodeVolumeDetachment are set

Reply:
we default `UnblockNodeVolumeDetachment`, whenever `VerifyNodeVolumeDetach` is true.